api/pkg/file/router.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 The SODA Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package file
+
+import (
+	"github.com/emicklei/go-restful"
+	"github.com/micro/go-micro/v2/client"
+)
+
+//RegisterRouter - route request to appropriate method
+func RegisterRouter(ws *restful.WebService) {
+	handler := NewAPIService(client.DefaultClient)
+	ws.Route(ws.GET("/{tenantId}/file/shares").To(handler.ListFileShare)).
```

Comment:
For the URL, my suggestion is as below:
/v1/tenantId/file-shares is for listing all file-shares
/v1/tenantId/file-shares?backend-id=*** is for listing file-shares bound to a specific backend-id
/v1/tenantId/file-shares/<file-shares-id> is for getting a specific file-share

Reply:
Thanks @sfzeng 
We just tried to make it consistent with the APIs we have for FS and Storage. Going ahead it will make sense for a unified API.(across cloud and premise)
As per our current sodafoundation/api:
![image](https://user-images.githubusercontent.com/48085413/85059422-3e490a00-b1c1-11ea-852d-2f0cb783045d.png)

Also, Cloud providers APIs have 'locations' in the path parameter and for gelato, backend is the location,so we have taken this approach.
IMO, let's go ahead with this approach. When we go for API standards for SODA then we may re-visit this and for that sense all the APIs across different core projects.
Hope it makes sense.